code/src/index.css

```diff
@@ -1,13 +1,223 @@
+@import url(//db.onlinewebfonts.com/c/e3619f7413527cff15063e007b666239?family=WrittenLies);
```

Comment:
This font is tricky to read and should have more weight and bigger size. Especially the footer is impossible to read in current size. This error shows in the inspect tool: Mixed Content: The page at 'https://book-labyrinth.netlify.app/' was loaded over HTTPS, but requested an insecure font 'http://db.onlinewebfonts.com/t/e3619f7413527cff15063e007b666239.woff2'. This request has been blocked; the content must be served over HTTPS.